frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/provision-source.tsx

```diff
@@ -70,17 +70,32 @@ export const ProvisionSourceComponent: React.FC<ProvisionSourceComponentProps> =
     const sources = iGet(provisionSourceField, 'sources');
 
     return (
-      <FormFieldRow field={provisionSourceField} fieldType={FormFieldType.SELECT}>
+      <FormFieldRow field={provisionSourceField} fieldType={FormFieldType.PF_SELECT}>
         <FormField>
-          <FormSelect onChange={(v) => onChange(VMSettingsField.PROVISION_SOURCE_TYPE, v)}>
-            <FormSelectPlaceholderOption
-              placeholder={getPlaceholder(VMSettingsField.PROVISION_SOURCE_TYPE)}
-              isDisabled={!!provisionSourceValue}
-            />
-            {(sources || []).map((source) => (
-              <FormSelectOption key={source} value={source} label={source} />
-            ))}
-          </FormSelect>
+          <FormPFSelect
+            menuAppendTo="parent"
+            selections={provisionSourceValue}
+            placeholderText={getPlaceholder(VMSettingsField.PROVISION_SOURCE_TYPE)}
+            onSelect={(e, v) => {
+              onChange(VMSettingsField.PROVISION_SOURCE_TYPE, v.toString());
+            }}
+          >
+            {(sources || [])
+              .sort(
+                (a, b) =>
+                  ProvisionSource.fromString(a).getOrder() -
```

Comment:
you can do the map first

Reply:
don't think so, the map returns a different array with `[<SelectOption>]`